syntax="proto3";

package beluga;

message RPCMsg{
    string service_name = 1;
    string method_name = 2;
    string request_data = 3;
}
